src/main/java/com/powsybl/openloadflow/network/extensions/AsymTransfo2W.java

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright (c) 2023, Jean-Baptiste Heyberger <jbheyberger at gmail.com>
+ * Copyright (c) 2023, Geoffroy Jamgotchian <geoffroy.jamgotchian at gmail.com>
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+package com.powsybl.openloadflow.network.extensions;
+
+import org.apache.commons.math3.complex.Complex;
+
+import java.util.Objects;
+
+/**
+ * @author Jean-Baptiste Heyberger <jbheyberger at gmail.com>
+ */
+public class AsymTransfo2W {
```

Comment:
Why this new extension compared to existing `TwoWindingsTransformerFortescue`? I see that you prefer using `z` here than `r` and `x` already present in the extension.